### DISCUSSION TITLE
Conditional Card Configuration does not allow greater and less than

### DISCUSSION THREAD

**Discussion by bogdanr** [home-assistant/frontend#9809] [Dashboard]
My use cases for this card need to specify greater than, instead of equal for showing conditional cards. Some examples when I want to show some cards:
* CPU usage is over 50%
* People detected greater than 0
* Temperature over 28°C
* Battery less than 10%

**Comment by nsmits** [comment]
Possible solution/workaround: create a binary sensor (true/false) for each condition with templating. See https://www.home-assistant.io/integrations/template/#state-based-template-sensors
